<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task3Window</class>
 <widget class="QDialog" name="Task3Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите новый элемент</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxList1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>77</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Первый</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxList2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>77</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Второй</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить элемент в список</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditBeforeSort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>181</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditAfterSort">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
